tests/RoundTrip/t_F_to_chiSq_base.0.hk

```diff
@@ -0,0 +1,14 @@
+def chiSq_iid(n nat, mean real, stdev prob):
+	q <~ plate _ of n: normal(mean,stdev)
+	return summate i from 0 to size(q): 
+		((q[i]-mean)/stdev)^2
+
+def stdChiSq(n nat):
+	chiSq_iid(n,0,1)
+
+def F(n1 nat, n2 nat):
+	X1 <~ stdChiSq(n1)
+	X2 <~ stdChiSq(n2)
+	return (X1*n2)/(X2*n1)
+
+F(1,∞)
```

Comment:
infinity may be accepted as a nat, but it really shouldn't. You can't substitute infinity into something and hope to have it work.  So this test is broken in non-repairable ways.  I suggest deleting this one.